public/js/admin.js

```diff
@@ -0,0 +1,306 @@
+            closeOnConfirm: false,
+            showLoaderOnConfirm: true,
+            },
+            function (){
+                document.location.href = "/dashboard/channels";
+            }
+        );
+}
+
+/**
+ * successCreateMessage modal message
+ */
+function successCreateMessage ()
+{
+    swal({
+            title: "Done!",
+            text: "New Channel has been added successfully",
+            type: "success",
+            showCancelButton: false,
+            closeOnConfirm: false,
+            showLoaderOnConfirm: true,
+            },
+            function (){
+                document.location.href = "/dashboard/channels";
+            }
+        );
+}
+
+/**
+ * successEditUser modal message
+ */
+function successEditUser ()
+{
+    swal({
+            title: "Done!",
+            text: "User has been successfully updated",
+            type: "success",
+            showCancelButton: false,
+            closeOnConfirm: false,
+            showLoaderOnConfirm: true,
+            },
+            function (){
+                document.location.href = "/dashboard/users";
+            }
+        );
+}
+
+/**
+ * successEditUser modal message
+ */
+function successInviteUser ()
+{
+    swal({
+            title: "Done!",
+            text: "Upgrade Invitation sent successfully",
+            type: "success",
+            showCancelButton: false,
+            closeOnConfirm: false,
+            showLoaderOnConfirm: true,
+            },
+            function (){
+                document.location.href = "/dashboard/users";
+            }
+        );
+}
+
+/**
+ * cancelDeleteMessage modal message
+ *
+ * @param  name
+ */
+function cancelDeleteMessage (name)
+{
+    swal("Cancelled", "Channel " + name + " is still available", "error");
+}
+
+/**
+ * errorMessage modal message
+ */
+function errorMessage ()
+{
+    swal("Error", "Error processing your request, Please try again!!!", "error");
+}
+
+/**
+ * errorInviteUser modal message
+ */
+function errorInviteUser ()
+{
+    swal("Error", "Invitation already sent, waiting confirmation from user", "error");
+}
+
+/**
+ * processAjax Proccess the ajax call
+ *
+ * @param  url
+ * @param  parameter
+ * @param  name
+ */
+function processAjax (url, parameter, name)
+{
+    $.post(url, parameter, function( data ){
+        if( data.status_code == 400){
+            return successDeleteMessage( name );
+        }
+        else if(data.status_code == 200){
+            return successUpdateMessage(name);
+        }
+        else if(data.status_code == 100){
+            return successCreateMessage(name);
+        }
+        else if (data.status_code == 600){
+            return successEditUser();
+        }
+        else if (data.status_code == 500){
+            return successInviteUser();
+        }
+        else if (data.status_code == 502){
+            return errorInviteUser();
+        }
+        else
+        {
+            errorMessage();
+        }
+    });
```

Comment:
**Code Quality - Nitpick**
I think a `switch` statement would be better here than the multiple `if-else-ifs`